### DISCUSSION TITLE
How do devs create new maps

### DISCUSSION THREAD

**Discussion by bookbug666** [LmeSzinc/AzurLaneAutoScript#1446] [Q&A] [answered]
Dev們都是怎麼創建新地圖的?總是會看到有不同伺服器的人問"怎麼不能使用Alas打XX圖?"(如#1445~~或是我自己也曾經不小心白目地去群裡問~~)
我有看wiki中map extract的說明但不得要點，關鍵的來源是哪裡呢?解包嗎?這是普通Alas使用者可以做到並貢獻給Alas社群的嗎

想說要是有簡單的方法就可以讓先開放地圖的伺服器玩家生成地圖測試，也方便其他伺服器上線後檢驗地圖正確性
如果不簡單或是這有技術門檻也可以補充在wiki關於地圖生成的部分，感謝開發組們

**Comment by LmeSzinc** [comment, marked as answer]
来源是 https://github.com/AzurLaneTools/AzurLaneLuaScripts ，这是一个自动解包仓库，map_extractor 用游戏的解包数据生成地图文件，但是自动生成的地图不一定能用，需要手动调整一下，这个比较依赖经验。
